src/Lumi/Components/Select.purs

```diff
@@ -484,7 +484,11 @@ styles = jss
               }
 
           , "&[data-multi=\"true\"]":
-              { "& lumi-select-input-selected-value":
+              { "& lumi-select-input":
+                  { height: important "unset"
```

Comment:
Do we want select inputs to grow like that? Perhaps only the multi-select variant?

Reply:
Yes, multi-selects need to. I've nested it under multi=true here.